### ISSUE TITLE
508-defect-2 [COGNITION, SCREEN READER]: Screen reader users SHOULD hear alerts when they appear on the page

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17814]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I want to know when alerts appear on the page so that I can hear any additional important content.

## Details

The information alertbox upon checking "I'd like the same office to do a review" is not announced to screen reader users which is an unequal experience to sighted users.

In this case, since the "I'd like the same..." checkbox will be moved to its own independent page, we may want to consider removing the alertbox in favor of redesigned content with the additional space.

## Acceptance Criteria

- [ ] The screen reader announces the alert
OR
- [ ] The alert is removed (recommended)

## Solution

@Mottie [has documented what this page will look like](https://github.com/department-of-veterans-affairs/vets-website/pull/15448) after being split from the select an issue page. @kevinstachura the recommendation here would be to remove the alert from this page. Given the additional space, you could consider modifying/rewording the content too.

e.g. 
A new reviewer will look at your case and consider it separately from your original decision. You can request to have the same office conduct your Higher-Level Review. 

If we cannot fulfill your request, we will let you know at the time we make a decision on your Higher-Level Review.

[Alternatively you could consider making the second line a hint accordion?]

## Screenshots or Trace Logs

<img width="770" alt="Screen Shot 2020-12-22 at 9 29 52 AM" src="https://user-images.githubusercontent.com/14154792/102899182-8b3fbe80-4438-11eb-8a86-60c164610348.png">

Removing the alert will also prevent this from showing up on mobile:

![image](https://user-images.githubusercontent.com/14154792/102900978-0ace8d00-443b-11eb-9eca-4abce23c59ce.png)

**Comment by joshkimux** [comment]
The alert no longer exists, so this can be closed.
